notes/views/test_rca.py

```
@@ -0,0 +1,45 @@
+        qs = qs[0]
+    return qs
+
+def return_data_value():
+    data = {
+        "name":"Preeti"
+    },
+    return data
+
+
+def get_story_text(story_info):
+    ds = get_qs_data(qs, data=story_info)
```

Ruff

Undefined name `qs`

(F821)